docs/deprecations.rst

```diff
@@ -253,8 +253,12 @@ Use instead::
     width, height = right - left, bottom - top
 
 Previously, the ``size`` methods returned a ``height`` that included the vertical
-offset of the text, while the new ``bbox`` methods explicitly distinguish this as a
-``top`` offset.
+offset of the text, while the new ``bbox`` methods distinguish this as a ``top``
+offset.
+
+.. image:: ./example/size_vs_bbox.png
+    :alt: Demonstration of size height vs bbox top and bottom
```

Comment:
Rather than saying what the image is, could you describe what the image conveys?

Imagine it read aloud to you when you can't see the image.

https://accessibility.huit.harvard.edu/describe-content-images

Reply:
I've pushed a commit.
> In bbox methods, top measures the vertical distance above the text, while bottom measures that plus the vertical distance of the text itself. In size methods, height also measures the vertical distance above the text plus the vertical distance of the text itself.

I don't really like it, but it was a struggle to explain the meaning without using the words 'height', 'top' and 'bottom' in the definitions of those terms themselves.

Reply:
Thanks! It can be a challenge to write good alt text.

I read it through and understood the difference without looking at the image. 👍